### PR TITLE
Refactored, reordered colours, colour amends

### PR DESCRIPTION
**Pull request by guywillis**
Improvements to legibility of the CSS
Theme colour amends

### CODE REVIEW

File: less/variables.less [hunk 1]
```diff
@@ -309,66 +309,66 @@
...
 
-@notify-popup-padding:@notify-popup-padding-top @notify-popup-padding-right @notify-popup-padding-bottom @notify-popup-padding-left;
+@notify-popup-padding: @notify-popup-padding-top @notify-popup-padding-right @notify-popup-padding-bottom @notify-popup-padding-left;
 
 
 // Fonts
 // =====
 
-@import url(//fonts.googleapis.com/css?family=Open+Sans:300,400,600);
+@import url(https://fonts.googleapis.com/css?family=Roboto:300,400,700);
```
Comment (moloko) [inline less/variables.less]: Please could you stick to a protocol-less URL like it was before? If you specify the protocol then you will get a secure/insecure content content issue - mainly in IE (which displays a confusing warning) but also I think Chrome too - if the content is run from server that uses a different protocol

**Comment by oliverfoster** [comment]
+1

**Comment by lc-thomasberger** [comment]
+1

**Comment by dancgray** [comment]
+1

**Pull request merged**